src/js/index.js

```diff
@@ -1,5 +1,167 @@
-(function () {
+/*global $ */
+'use strict';
 
-  console.log('WOLOX');
+//Manejo de animaciones cuando abre el sitio
+function openBars(time){
+	$('.copyright').animate({
+		bottom: '+=200'
+	},time,function(){});
 
-})();
+	$('header').animate({
+		marginTop: '+=400'
+	},time,function(){});
+}
+
+var slide = 1;
+
+//Manejo del slideshow
+$('.backcontrols').hover(function(){
+	if ($('.copyright').css('bottom') ==='-200px'){
+		openBars(1000);
+	}
+},function(){});
+
+
+function pasar(slide){
+	$('body').css('background-image', 'url("../assets/slide' + slide + '.jpg")');
+}
+
+
+var paso;
```

Comment:
leave an enter after the var declaration, also declare all the variables at the begining of each function.